lib/models/ColdStorageActionQueueEntry.js

```diff
@@ -0,0 +1,91 @@
+const joi = require('joi');
+
+const entrySchema = joi.object({
+    op: joi.string().valid('restore', 'delete', 'archive').required(),
+    bucketName: joi.string().required(),
+    objectKey: joi.string().required(),
+    objectVersion: joi.string().required(),
+    accountId: joi.string().required(),
+    archiveInfo: joi.object({
+        archiveId: joi.string().required(),
+        archiveVersion: joi.string().required(),
+    }).required(),
```

Comment:
Please make `archiveId` opaque. It should never be looked into, as it's handled by the TLP implementation and could change depending on backend (tape, azure archive, glacier, ...).